resources/views/mail/contact.blade.php

```diff
@@ -0,0 +1,8 @@
+<h2>{{ $contactForm->firstName }} {{ $contactForm->lastName }} has said the following:</h2>
```

Comment:
Please check if the headers are correct now, I've changed them from a h1 and h2 into a h2 and h3.

Reply:
fine by me